axlearn/experiments/text/gpt/fuji.py

```diff
@@ -2,7 +2,7 @@
 
 """Utilities to set up the 'fuji' GPT model trainer configs.
 
-Architecture names follow apple varieties: fuji, gala, honeycrisp, etc.
+Architecture names follow apple varieties: Fuji, Gala, Honeycrisp, etc.
```

Comment:
Could you confirm that this PR doesn't change the step times for the Fuji models?